relentless/core/variable.py

```diff
@@ -258,6 +261,8 @@ def __init__(self, *vardicts, **kwvars):
             raise AttributeError('No attributes specified for DependentVariable.')
 
         for k,v in attrs.items():
+            if np.isscalar(v):
```

Comment:
We also need to catch this in the `__setattr__` method below. I think probably what we should do is change `_asert_variable` to a `_make_variable`, and have it do this check rather than raise an error. This would mean you don't need to change the `__init__` or `__setattr__` methods, other than using the new method call.